frontend/.eslintcache

```diff
@@ -1 +1 @@
\ No newline at end of file
+[{"/home/genuineeek/Workspace/Github/7/frontend/src/index.js":"1","/home/genuineeek/Workspace/Github/7/frontend/src/reportWebVitals.js":"2","/home/genuineeek/Workspace/Github/7/frontend/src/App.js":"3","/home/genuineeek/Workspace/Github/7/frontend/src/components/Pages/Settings.js":"4","/home/genuineeek/Workspace/Github/7/frontend/src/components/Pages/Home.js":"5"},{"size":500,"mtime":1611438784360,"results":"6","hashOfConfig":"7"},{"size":362,"mtime":1611438784360,"results":"8","hashOfConfig":"7"},{"size":1374,"mtime":1611473185654,"results":"9","hashOfConfig":"7"},{"size":1021,"mtime":1611478146177,"results":"10","hashOfConfig":"7"},{"size":353,"mtime":1611473392260,"results":"11","hashOfConfig":"7"},{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},"151wbru",{"filePath":"15","messages":"16","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"17","messages":"18","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"19","messages":"20","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},"/home/genuineeek/Workspace/Github/7/frontend/src/index.js",[],["23","24"],"/home/genuineeek/Workspace/Github/7/frontend/src/reportWebVitals.js",[],"/home/genuineeek/Workspace/Github/7/frontend/src/App.js",[],"/home/genuineeek/Workspace/Github/7/frontend/src/components/Pages/Settings.js",[],"/home/genuineeek/Workspace/Github/7/frontend/src/components/Pages/Home.js",[],{"ruleId":"25","replacedBy":"26"},{"ruleId":"27","replacedBy":"28"},"no-native-reassign",["29"],"no-negated-in-lhs",["30"],"no-global-assign","no-unsafe-negation"]
```

Comment:
We might not need this cache file on source control, consider creating a .gitignore for this.